scipy/io/matlab/tests/test_mio.py

```diff
@@ -1065,22 +1065,22 @@ def test_fieldnames():
 
 def test_loadmat_varnames():
     # Test that we can get just one variable from a mat file using loadmat
-    mat5_sys_names = ['__globals__',
-                      '__header__',
-                      '__version__']
+    mat5_sys_names = {'__globals__', '__header__', '__version__'}
     for eg_file, sys_v_names in (
-        (pjoin(test_data_path, 'testmulti_4.2c_SOL2.mat'), []), (pjoin(
-            test_data_path, 'testmulti_7.4_GLNX86.mat'), mat5_sys_names)):
+            (pjoin(test_data_path, 'testmulti_4.2c_SOL2.mat'), set()),
+            (pjoin(test_data_path, 'testmulti_7.4_GLNX86.mat'), mat5_sys_names)):
         vars = loadmat(eg_file)
-        assert_equal(set(vars.keys()), set(['a', 'theta'] + sys_v_names))
+        assert_equal(set(vars.keys()), {'a', 'theta'} | sys_v_names)
```

Comment:
Set literals do not work on Python 2.6.  To make it clearer what the commit is doing, I suggest you revert these changes and just leave the test at the end.